src/components/CreateMessageCheck.tsx

```diff
@@ -94,44 +97,54 @@ const sendButtonStyle = css`
  * 作成したメッセージを確認する画面を返す
  */
 
-export const CreateMessageCheck = () => {
+export const CreateMessageCheck = async (ctx: Context) => {
+  const id: string = ctx.req.query("id") || "";
```

Comment:
```suggestion
export const CreateMessageCheck = async (
  props: PropsWithChildren<{ id: string }>,
) => {
  const id: string = props.id;
```

`route.tsx`の変更を入れるとこっちもこんな感じにする必要がありそう

Reply:
修正しましたー
[141cb03](https://github.com/jigintern/online-2024-e/pull/47/commits/141cb038c52d16f38309ab1041095791c7cc0836)